.test-infra/jenkins/job_LoadTests_ParDo_Flink_Python.groovy

```diff
@@ -161,12 +164,13 @@ def streamingScenarios = { datasetName ->
       test           : 'apache_beam.testing.load_tests.pardo_test',
       runner         : CommonTestProperties.Runner.PORTABLE,
       pipelineOptions: [
-        job_name             : 'load-tests-python-flink-streaming-pardo-5-' + now,
+        job_name             : 'load-tests-python-flink-streaming-pardo-1-' + now,
```

Comment:
I have mixed feelings about it. This test is not `load-tests-python-flink-batch-pardo-1` but on streaming. There are more differences between them: batch-pardo-1 uses 10 iterations, this test uses 5 iterations. 0 counters in batch-pardo-1 vs. 3 counters right here. Because of that, I think we should stay with the previous job_name: `load-tests-python-flink-streaming-pardo-5`.

The general idea behind load tests is that we run the same configuration on different runners, in different SDKs and in different mode (batch or streaming). Grafana dashboards for load tests were designed with that convention in mind. If you choose `java` and `streaming` from the list, Grafana will pull data from these measurements: `java_streaming_pardo_1`, `java_streaming_pardo_2` and so. Your streaming tests are a bit problematic, because they are not being run on Dataflow and batch. Also, they have no Java counterpart.

That being said, I think about two solutions:
1) Add more charts. We would end up with a total of six charts. The fifth and the sixth chart would be empty in most cases (for Java and for batch).
2) Create a separate, more specific version of dashboard just for these two tests (streaming-pardo-5 and streaming-pardo-6). Leave "ParDo Load Tests" dashboard intact.

@mxm What do you think?

Reply:
Note, this is just the job name. More important is the table we are writing to further down. Unfortunately, the Grafana setup forces me to do that. I would rather not change this but the Grafana setup is very inflexible and in this regard a regression from the old framework we used: https://apache-beam-testing.appspot.com/explore?dashboard=5751884853805056

>Your streaming tests are a bit problematic, because they are not being run on Dataflow and batch.

I don't fully understand your point to be honest, in order for the dropdown menus to work properly, i.e. choosing `SDK` and the `mode` (batch/streaming), this change is required because the table name is composed of `$sdk_$mode_`. The test parameters looked identical to me for Dataflow/Flink. If the iterations don't match, we can adjust that. The input is already the same.

Adding more charts would be another option, though we completely lose the idea of being able to compare Runner performance. We have to remove the streaming dropdown and just add one chart per streaming and batch run. I think that is the best option. It gives us a bit more flexibility.

Reply:
@kamilwu If you agree, I'd remove the streaming/batch dropdown and just add a new chart for the streaming mode. I suppose that is a better migration path because there are no other streaming load tests at the moment.

Reply:
> there are no other streaming load tests at the moment.

Not quite. At the moment, we have streaming load tests for Java (Dataflow only). Apart from that, I'm investigating running other Python load tests (ParDo 1, 2, 3 and 4) in streaming mode too.

Reply:
@kkucharc is actually working on streaming load tests for Python on Dataflow and she's already prepared a PR: https://github.com/apache/beam/pull/12435. We would like to show metrics from these new tests too.

Reply:
Good to hear. Still, there is no streaming data in the dashboards yet :)

So do you agree to add new dashboards for the streaming tests and removing the selector for batch/streaming?

Reply:
> If the iterations don't match, we can adjust that. The input is already the same.

@mxm Do you think it is possible to adjust those parameters so that `pardo-5` can become `pardo-1` and `pardo-6` can become `pardo-2`, `pardo-3` or `pardo-4`? The main advantage of this solution is that we wouldn't have to modify dashboards at all. The old version would just work.

Reply:
If not, then I'm fine with adding new charts (I suppose you'd meant "chart", "dashboard" is a different kind of thing) and removing the selector for batch/streaming.

Reply:
> @mxm Do you think it is possible to adjust those parameters so that `pardo-5` can become `pardo-1` and `pardo-6` can become `pardo-2`, `pardo-3` or `pardo-4`? The main advantage of this solution is that we wouldn't have to modify dashboards at all. The old version would just work.

That was the original idea in this PR which you I understood you didn't like. pardo_5 became pardo_1. As for pardo_6, that's not possible because it measures the checkpoint duration and should be a separate panel.

> If not, then I'm fine with adding new charts (I suppose you'd meant "chart", "dashboard" is a different kind of thing) and removing the selector for batch/streaming.

Yes, I meant panel, corrected above.

Reply:
> As for pardo_6, that's not possible because it measures the checkpoint duration and should be a separate panel.

I see. Then, let's do the opposite way (adding new charts and removing the selector). Thank you.

Reply:
I went the other route you suggested and adjusted the parameters for the load tests. Adding more panels seemed like a good idea but it also adds significant noise to the dashboard.

Reply:
As for the latency / checkpoint duration. I think they are good panels to have which are applicable to many runners. I'd like to keep them where they are so we can follow the performance regression guidelines in the release guide.